### ISSUE TITLE
Import study: parser error : Input is not proper UTF-8

### ISSUE THREAD

**Issue by PEvelynNguyen** [qualintitative/egoweb#20]
So I got the site up and running, and even succeed in importing some studies from AWS to Linode. However, I got stuck with 3 studies. I tried importing the .study files as well as using the "send study to server" feature many times. The error read: 

" 
Error 500
simplexml_load_file(): /tmp/phpSKdZlI:213: parser error : Input is not proper UTF-8, indicate encoding ! Bytes: 0xE6 0x43 0x61 0x68
"

Any idea how to fix it? I don't know why it worked for some studies and not for some. 

Thanks,
Evelyn

**Comment by qualintitative** [comment]
Just to confirm, did you convert the old study first before exporting?  I have forgotten to do that in the past for studies that I thought were already converted, so I thought I'd mention it just in case.

**Comment by PEvelynNguyen** [comment]
Yes, I did. Just checked again, there's no "convert" button in any of these studies.

**Comment by qualintitative** [comment]
I'll have to ask @zhangster to interpret this error when he is back from a vacation next week.
